### PR TITLE
Aizad/Market Restructure improvement: Arrange the subgroup name on mobile view

### PR DESCRIPTION
**Pull request by aizad-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by suisin-deriv — DISMISSED** [review]
LGTM

File: src/sass/app/components/market.scss [hunk 1]
```diff
@@ -315,8 +315,26 @@
...
                     .label {
                         font-size: 20px;
-                        padding: 16px 0px;
-                        font-weight: 700;
+                        padding: 16px 0;
+                        font-weight: bold;
+                        flex: 1 0 auto;
+                        @media (min-width: 320px) and (max-width: 767px) {
+                            font-size: 20px;
+                        }
```
Comment (matin-deriv) [inline src/sass/app/components/market.scss]: Why is this needed? `font-size` is already defined at line 317

**Review by farzin-deriv — DISMISSED** [review]

File: src/sass/app/components/market.scss (hunk 1 above)
Comment (aizad-deriv) [reply src/sass/app/components/market.scss]: When the screen is resize based on the rule it applies the font size of `20px` for the mobile and tablet view.

**Review by matin-deriv — DISMISSED** [review]

**Review by ameerul-deriv — APPROVED** [review]
LGTM

**Review by jim-deriv — APPROVED** [review]
